admin/views/pupil-pin/select-official-or-try-it-out-pins.ejs

```diff
@@ -9,20 +9,20 @@
             </h1>
 
             <p>
-              Pupils will need a PIN and a school password to access the MTC. The link for the pupil to start the check
-              is
-              <a class="govuk-link" target="_blank" rel="noopener noreferrer" href="<%= pupilAppURL %>">
+              Pupils will need a school password and individual PIN to sign into the [TODO:CHECK_TYPE] check.
```

Comment:
Flagging TODO item for a later review, when this PR is finalised.

Reply:
removed :)